packages/faustwp-core/src/hooks/useAuth.tsx

```diff
@@ -114,9 +137,54 @@ export function useAuth(_config?: UseAuthConfig) {
     }, 200);
   }, [isReady, isAuthenticated, loginUrl, config]);
 
+  /**
+   * Expose viewer options to the toolbar if the user is authenticated.
+   */
+  useEffect(() => {
+    if (config.skip === true) {
+      return;
+    }
+
+    if (isAuthenticated !== true) {
+      return;
+    }
+
+    (async () => {
+      const client = getApolloAuthClient();
+
+      const { data } = await client.query({
+        query: gql`
+          query Viewer {
```

Comment:
We should call this query something unique, in case there is another query the user created called `Viewer`.

Maybe we call it something like `GetFaustViewer`?

Reply:
@TeresaGobble Looks like the tests are failing again due to the change from `Viewer` to `GetFaustViewer`. You'll have to update the mock URL to reflect this.

Reply:
Thank you @blakewilson. The e2e test failed again here after changes, but rerunning it again fixed it. Should be good for a final look.